buildenv/jenkins/variables/defaults.yml

```diff
@@ -163,11 +163,11 @@ boot_jdk_default:
       17: '17'
       19: '18'
       20: '18'
-      next: '18'
+      next: 'https://api.adoptium.net/v3/binary/latest/20/ga/${os}/${arch}/jdk/hotspot/normal/eclipse?project=jdk'
```

Comment:
Does this work? I think you need to put the version here and the url under `url:`

Reply:
I think you're right, perhaps this section should look like this?
```
boot_jdk_default:
  boot_jdk:
    location:
      all: '${HOME}/bootjdks'
    version:
      8: '8'
      11: '11'
      17: '17'
      19: '18'
      20: '18'
      next: '20'
    dir_strip:
      all: '1'
    url:
      all: 'https://api.adoptopenjdk.net/v3/binary/latest/${bootJDKVersion}/ga/${os}/${arch}/jdk/openj9/normal/adoptopenjdk?project=jdk'
      next: 'https://api.adoptium.net/v3/binary/latest/${bootJDKVersion}/ga/${os}/${arch}/jdk/hotspot/normal/eclipse?project=jdk'
```
@AdamBrousseau ?

Reply:
Looks good to me, I believe we've done something like that before.

Reply:
Updated.